### DISCUSSION TITLE
Sprite flags should be editable on 2bpp and 1bpp

### DISCUSSION THREAD

**Discussion by Weeppiko** [nesbox/TIC-80#1555] [General]
Being able to edit the sprite flags only on 4bpp mode is a bit counter-intuitive if I want to mainly use 2bpp for a project. My usage case would be to use both flags and remap function to change colors for tiles when drawing it, but I'm afraid that it becomes more and more confusing to set the right flags on the right tile since they just show as corrupted graphics on 4bpp mode.

**Comment by Morokiane** [comment]
Agree with this. I want to use the 2-bit and 1-bit modes more, but still need all the functionality that is available in 4-bit.

**Comment by nesbox** [comment]
We can add flags to the lower bit modes but it won't be 8 flags per sprite, because we only have 512 bytes for the flags in the [RAM](https://github.com/nesbox/TIC-80/wiki/ram). So, for the 2-bit mode, we can only show 4 flags per sprite and 2 flags for the 1-bit mode to fit 512 bytes.

**Reply by Weeppiko** [reply]
Yes, that was what I expected it to be in the first place. It's a good tradeoff deciding to use more pages for sprites by having less resources for each sprite, but having no flags is a huge barrier for this due how useful they can be.
